analytics-listeners/src/main/java/org/exoplatform/analytics/listener/social/AnalyticsActivityTagsListener.java

```diff
@@ -73,6 +73,7 @@ private void addEventStatistic(Set<TagName> tagNames, String activityId) {
     statisticData.setUserId(Long.parseLong(activity.getUserId()));
     statisticData.addParameter("username", username);
     statisticData.addParameter("type", objectType);
+    statisticData.addParameter("dataType", objectType + "_TYPE");
```

Comment:
```suggestion
    statisticData.addParameter("dataType", StringUtils.lowercase(objectType));
```
Same here, it's better to conserve the exact same value as objectType but in lowercase (for case insensitive)

Reply:
Done!